app/src/main/res/layout/fragment_member_bottom_sheet.xml

```diff
@@ -33,15 +34,26 @@
             android:textColor="@color/white"
             tools:text="Ronald Perkins" />
 
+        <ImageView
+            android:id="@+id/user_status"
+            android:layout_width="10dp"
+            android:layout_height="10dp"
+            android:layout_marginStart="5dp"
+            android:background="@drawable/user_status_white"
+            app:layout_constraintBottom_toBottomOf="@id/text_bottom_sheet_member_name"
+            app:layout_constraintStart_toEndOf="@id/text_bottom_sheet_member_name"
+            app:layout_constraintTop_toTopOf="@id/text_bottom_sheet_member_name" />
+
         <TextView
```

Comment:
Could you change the side of how the user status is shown? I mean, instead of showing it on the right side, show on the left side of the user name.

Why? This will be aligned with how the user status will be show on the chat list screen.

Also, you don't need to use an `ImageView` to do that, use a `drawableleft` with a `drawablePadding`

https://developer.android.com/reference/android/widget/TextView.html#attr_android:drawableLeft

Reply:
Cool @filipedelimabrito , I'll make the changes .

Reply:
But if I give a `drawableStart`, then the username and name won't be aligned with each other @filipedelimabrito.

Reply:
@filipedelimabrito ^^

Reply:
@filipedelimabrito shall I make the changes?

Reply:
@aniketsingh03 You can adjust padding or margin of username and name to make align perfectly

Reply:
Yeah, that's fine @Shailesh351, but then we would need to provide `paddingStart` to the other content too like the **utcOffset** and content below that to make it look appropriate. @filipedelimabrito what do you think?